### PR TITLE
Event functionality

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #108

### CODE REVIEW

File: src/routes/event.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const { mockEvents } = require('../../database/events_schema.js');
+
+module.exports = (req, res) => {
+  const id = req.params.id;
```
Comment (esraajb) [inline src/routes/event.js]: lines 4 to 9 is what im talking about, take this code and put it inside a function in a different file

Comment (Karyum) [inline src/routes/event.js]: is because of readability reasons or testing reasons???

Comment (esraajb) [inline src/routes/event.js]: Both

Comment (Karyum) [inline src/routes/event.js]: i think its only going to make harder if its for testing im going to push all the tests that i did for the routes soon, also for the readability it doesn't make any sense to split the files cause the one who is going to read the code is just going to have to jump between 2 files its bad DX ( developer experience :stuck_out_tongue_winking_eye: )

Comment (esraajb) [inline src/routes/event.js]: Benefits of splitting your code into small functions:
1) easier to spot bugs
2) easier to test
3) easier for other developers to read
4) easier for future maintenance/improvements  
...

I suggest that you do that

**Comment by esraajb** [comment]
@Karyum @Suuuuha  codecov is failing

**Comment by Karyum** [comment]
@esraajb it's passing now :maple_leaf:

**Pull request merged**